frontend/src/components/Table/ColumnFilter.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import { ColumnInstanceWithProps } from '.';
+import styled from 'styled-components';
+import { Form } from 'react-bootstrap';
+import ClickAwayListener from 'react-click-away-listener';
+import { getIn, useFormikContext } from 'formik';
+import clsx from 'classnames';
+import { FiFilter } from 'react-icons/fi';
+import { FaFilter } from 'react-icons/fa';
+
+interface IColumnFilterProps {
+  column: ColumnInstanceWithProps<any>;
+  onFilter: (values: any) => void;
+}
+
+const Wrapper = styled('div')`
+  display: flex;
+  align-items: center;
+  justify-items: center;
+  position: relative;
+  font-size: 12px;
+`;
+
+const InputContainer = styled('div')`
+  position: absolute;
+  bottom: -55px;
```

Comment:
Does this work ok if the columns get resized on smaller screens?

Reply:
yes it works